products_service/docker-entrypoint.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
-until psql -h host.docker.internal -p 5433 -U yugabyte -c 'show server_version' ; do sleep 1 ; done 2>/dev/null
+until psql -h host.docker.internal -p 5433 -U yugabyte -c 'show server_version' ; do
+    echo 'Waiting for yb-tserver (ysql) to be ready ...'; sleep 1
+done  2>/dev/null
+
 [ "$NODE_APP_INSTANCE" = los-angeles ] && npm run seed
-npm start
\ No newline at end of file
+
+if [[ $NODE_ENV == development ]]; then
+    exec node /app/node_modules/.bin/nodemon index.js
```

Comment:
should this read 
`exec /app/node_modules/.bin/nodemon index.js`
?

does `node` need to precede the path to `nodemon`?

Reply:
Either way works~ `/app/node_modules/.bin/nodemon` is just a node script with a `#!/usr/bin/env node` at the top.